app/src/components/ResourceInfoCard.tsx

```diff
@@ -118,6 +118,18 @@ export default function ResourceInfoCard({ resourceInfo, authoring }: ResourceIn
                 </ActionIcon>
               </Tooltip>
             </Link>
+            <Link
+              href={{
+                pathname: `/review/${resourceInfo.resourceType}/${resourceInfo.id}`,
+                query: { authoring: `${authoring}` }
+              }}
+            >
+              <Tooltip label={authoring ? 'Review Draft Artifact' : 'Review Artifact'} openDelay={1000}>
+                <ActionIcon radius="md" size="md" variant="subtle" color="gray">
```

Comment:
Maybe you can change the color of this so that it is not the same as the edit icon? They don't look super distinct at the moment:
![image](https://github.com/projecttacoma/measure-repository/assets/30158156/7cc8d328-bece-49d0-986d-a96aad313b60)

Maybe one can be blue? Maybe that will look bad though, play around with what you think looks best!